routes/assign-bus-routes/assign.js

```diff
@@ -0,0 +1,36 @@
+import { Router } from 'express';
+import {
+	Assign,
+} from '../../controllers/busRouteAssignController.js';
+
+const assignRouter = Router();
+
+
+
+/**
+ * @swagger
+ * /api/v1/assign/bus-to-route/{plate}/{code}:
+ *  post:
+ *    summary: assign bus to route
+ *    tags:
+ *      - Bus to route
+ *    description: fill in your plate nuber and route code
+ *    parameters:
+ *      - in: path			
+ *        name: code
+ *        description: 
+ *      - in: path			
+ *        name: plate
+ *        description: 
+ *    responses:
+ *      200:
+ *        description: successful
+ * 
+ */
+
+assignRouter.post(
+	'/bus-to-route/:plate/:code',
+	Assign
```

Comment:
It will be perfect if you protect this route.

Reply:
noted! hope now it is good